aspnetcore/toc.yml

```diff
@@ -1005,6 +998,14 @@
       uid: fundamentals/troubleshoot-aspnet-core-localization
 - name: Advanced
   items:
+    - name: Model binding
+      uid: mvc/models/model-binding
+    - name: Model validation
+      uid: mvc/models/validation
+    - name: Custom model binding
```

Comment:
I suggest swapping the positions of **Model validation** and **Custom model binding**. Then the 2 model binding docs are grouped together.